### PR TITLE
Use a rtd conf file

### LINKED ISSUE #102: Use a configuration file for Read the Docs
I notice this comment here https://github.com/theacodes/nox/blob/c64cc8e43ca41ef1e9eae71073f0b696538e3dc3/nox.py#L65

If you want to install using pip on rtd, you can, but you need to use a configuration file https://docs.readthedocs.io/en/latest/yaml-config.html#python-pip-install

### PR DESCRIPTION
**Pull request by stsewd**
I just tried to keep the current behavior, hope it's fine :)

Close #102

### CODE REVIEW

**Comment by stsewd** [comment]
Here is a sample build on rtd https://readthedocs.org/projects/nox-stsewd-test/builds/7676823/

**Comment by stsewd** [comment]
One job on travis timed out :(

File: .readthedocs.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+formats: ['htmlzip']
```
Comment (dhermes) [inline .readthedocs.yml]: Any reason for this? HTML and JSON will always be built by default so I use:

```yaml
# Don't build any extra formats, HTML and JSON will
# always be built by default.
formats:
  - none

requirements_file: docs/requirements.txt

build:
  image: latest

python:
  pip_install: true
  setup_py_install: false
  version: 3.6
```

whenever I have a RTD config.

Comment (stsewd) [reply .readthedocs.yml]: I include this format because that is how the docs are built right now, but we can drop it

Comment (theacodes) [reply .readthedocs.yml]: Let's do that.

Comment (stsewd) [reply .readthedocs.yml]: Done!

**Pull request merged**